rundown/resources/team.py

```diff
@@ -0,0 +1,23 @@
+from pydantic import BaseModel
+
+
+class Team(BaseModel):
+    team_id: int
+    name: str
+    mascot: str
+    abbreviation: str
+
+
+class TeamNormalized(Team):
```

Comment:
Can we just move all of these to a `Team` model (but using the `teams_normalized` attribute of the response?)

Reply:
Yeah that makes sense. I will do that

Reply:
I realized that the reason I broke team into `Team` and `TeamNormalized` is that `GET /sports/<sport-id>/teams` only has the data in `Team`, but all the events routes also include the data in `TeamNormalized`.

Reply:
Ok, that makes sense and works for me.